output/forms.eslint.txt

```diff
@@ -13,5 +13,27 @@ The react/require-extension rule is deprecated. Please use the import/extensions
 /home/travis/build/Talend/ui/packages/forms/src/UIForm/fields/Radios/Radios.component.js
   23:7  error  Form controls using a label to identify them must be programmatically associated with the control using htmlFor  jsx-a11y/label-has-for
 
-✖ 3 problems (3 errors, 0 warnings)
+/home/travis/build/Talend/ui/packages/forms/src/UIForm/fieldsets/Array/ArrayItem.component.js
+  46:64  error  A space is required before closing bracket  react/jsx-space-before-closing
 
+/home/travis/build/Talend/ui/packages/forms/src/widgets/ColumnsWidget/ColumnsWidget.js
+  38:3  error  Prop type `object` is forbidden  react/forbid-prop-types
+  39:3  error  Prop type `object` is forbidden  react/forbid-prop-types
+  40:3  error  Prop type `object` is forbidden  react/forbid-prop-types
+  78:3  error  Prop type `object` is forbidden  react/forbid-prop-types
+  79:3  error  Prop type `object` is forbidden  react/forbid-prop-types
+
+/home/travis/build/Talend/ui/packages/forms/src/widgets/KeyValueWidget/KeyValueWidget.js
+  72:3  error  Prop type `object` is forbidden  react/forbid-prop-types
+  73:3  error  Prop type `object` is forbidden  react/forbid-prop-types
+
+/home/travis/build/Talend/ui/packages/forms/src/widgets/ListViewWidget/ListViewWidget.js
+  123:3  error  Prop type `any` is forbidden  react/forbid-prop-types
+
+/home/travis/build/Talend/ui/packages/forms/src/widgets/TabsWidget.js
+  56:3  error  Prop type `object` is forbidden  react/forbid-prop-types
+  67:3  error  Prop type `object` is forbidden  react/forbid-prop-types
+  68:3  error  Prop type `object` is forbidden  react/forbid-prop-types
+  69:3  error  Prop type `object` is forbidden  react/forbid-prop-types
```

Comment:
could remove this lint from our official lint ?
We have so many case where shape is not appropriate..

Reply:
This is weird, we only have 1 eslintrc now at root and we have `"react/forbid-prop-types": [0]`
Run `yarn lint:es` locally, and I have not those issues.
Let's wait for next travis run

Reply:
The rule is already removed from our tool repo

Reply:
Hmm ok, seems to have a ci issue. I removed the linter results, the ci should recreate them, but they are still removed ...